### PR TITLE
live: Add dvclive logger and `DVCLIVE_LOGLEVEL`.

### LINKED ISSUE #206: `log`: Verbose mode
It would be nice to have a way to see what dvclive is doing instead of having it operate silently, so that stdout/loggers could optionally have info about when dvclive calls are made and what they are logging. This would be especially helpful in debugging.

### PR DESCRIPTION
**Pull request by daavoo**
Closes #206

* [X] ❗ I have followed the [Contributing to DVCLive](https://github.com/iterative/dvclive/blob/master/CONTRIBUTING.md) guide.

* [ ] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here.

### CODE REVIEW

**Comment by pared** [comment]
LGTM. Tested on a actual repo using `matplotlib` too. Seems like setting the `DEBUG` level will result in logs from this lib too. Not sure how to approach that. Probably we should not propagate it down to other used libraries.

**Comment by daavoo** [comment]
> LGTM. Tested on a actual repo using `matplotlib` too. Seems like setting the `DEBUG` level will result in logs from this lib too. Not sure how to approach that. Probably we should not propagate it down to other used libraries.

Updated P.R. It should no longer affect other libs, could you confirm?

File: dvclive/live.py [hunk 1]
```diff
@@ -46,7 +47,7 @@ def __init__(
 
         if self._report is not None:
             out = Path(self.html_path).resolve()
-            print(f"Report will be saved at {out}")
+            logger.info(f"Report will be saved at {out}")
```
Comment (daavoo) [inline dvclive/live.py:50]: Can be a good time to address https://github.com/iterative/dvclive/pull/247#issuecomment-1116425371 . Any suggestions?

> Report path (if generated): 

^ Reads fine to me

**Comment by pared** [comment]
I can confirm that now I see only `dvclive` logs for `DEBUG` option.

**Review by pared — APPROVED** [review]

File: dvclive/live.py (hunk 1 above)
Comment (dberenbaum) [reply dvclive/live.py:50]: Yup, I think that's fine.

**Review by dberenbaum — APPROVED** [review]

**Pull request merged**